FogCreek/Glitch-Community
# Boggy ice: refactor whitelisted-domain & add-team-user-popup

## Links
* Remix link: https://boggy-ice.glitch.me

## GIF/Screenshots:
add a domain (with favicon)
![Screen Shot 2019-05-13 at 12 52 01 PM](https://user-images.githubusercontent.com/938722/57640577-e2a15b80-7580-11e9-83f3-b6c4832adb84.png)

add a domain (no favicon)
![Screen Shot 2019-05-13 at 1 35 45 PM](https://user-images.githubusercontent.com/938722/57641914-2b0e4880-7584-11e9-9b00-dd72eef590f4.png)

invite by email
![Screen Shot 2019-05-13 at 1 28 46 PM](https://user-images.githubusercontent.com/938722/57641479-19787100-7583-11e9-8f5e-25ceeb83cf3b.png)

domain listed in project
![Screen Shot 2019-05-13 at 12 52 21 PM](https://user-images.githubusercontent.com/938722/57640578-e2a15b80-7580-11e9-96df-3d20c472272d.png)

remove domain from project
![Screen Shot 2019-05-13 at 12 58 34 PM](https://user-images.githubusercontent.com/938722/57640580-e2a15b80-7580-11e9-880e-84234deacb08.png)

## Changes:
* componentize whitelisted domain icon
* inline elements of user-result-item into add-team-user-popup
* fix styles for adding whitelisted domain

## How To Test:
* visit https://boggy-ice.glitch.me/secret and enable "email invites"
* create a team
* click add and verify that you can add users by login or email
* verify that whitelisted domains works with and without favicons (e.g. using glitch.com and example.com)
